estate/models/estate_property.py

```diff
@@ -0,0 +1,133 @@
+from odoo import models, fields, api
+from odoo.exceptions import UserError, ValidationError
+from odoo.tools.float_utils import float_compare
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Estate property"
+    _order = "id desc"
+
+    name = fields.Char(required=True, string="Title")
+    description = fields.Text()
+
+    postcode = fields.Char()
+    date_availability = fields.Datetime(
+        copy=False,
+        default=lambda self: fields.Datetime.add(fields.Datetime.today(), months=3),
+        string="Available From",
```

Comment:
I would have done it like this (or even calling a compute method to have a single line property) but it's up to you:
```suggestion
    date_availability = fields.Datetime(
        "Available From", copy=False,
        default=lambda self: fields.Datetime.add(fields.Datetime.today(), months=3),
```